index.html

```diff
@@ -70,141 +75,149 @@ <h2 class="yoda">
-            personalized reads; no accounts or <br />
-            sign-ups required.
-          </p>
-          <ul class="grid-list2">
-            <li class="list2">HTML</li>
-            <li class="list2">CSS</li>
-            <li class="list2">Javascript</li>
-          </ul>
-          <h3 class="seeproject">See Project</h3>
+          <img src="Snapshoot Portfolio.svg" alt="" class="gridpicture3" />
+          <div class="grid-container-text">
+            <h2 class="tonic">Facebook 360</h2>
+            <ul class="grid-list">
+              <li class="canopy-list">FACEBOOK</li>
+              <li class="back-list">Full Stack Dev</li>
+              <li class="year-list">2015</li>
+            </ul>
+            <p class="gridtext">
+              Exploring the future of media in Facebook's first Virtual Reality
+              <br />
+              app; a place to discover and enjoy 360 photos and videos on <br />
+              Gear VR.
+            </p>
+            <ul class="grid-list2">
+              <li class="list2">html</li>
+              <li class="list2">Ruby on rails</li>
+              <li class="list2">CSS</li>
+              <li class="list2">Javascript</li>
+            </ul>
+            <h3 class="seeproject">See Project</h3>
+          </div>
         </div>
 
         <div class="grid-container">
-          <img src="Snapshoot Portfolio(3).svg" alt="" class="gridpicture" />
-          <h2 class="tonic">
-            Multi-Post <br />
-            Stories
-          </h2>
-          <ul class="grid-list">
-            <li class="canopy-list">CANOPY</li>
-            <li class="back-list">Back End Dev</li>
-            <li class="year-list">2015</li>
-          </ul>
-          <p class="gridtext">
-            A daily selection of privately <br />
-            personalized reads; no accounts or <br />
-            sign-ups required.
-          </p>
-          <ul class="grid-list2">
-            <li class="list2">HTML</li>
-            <li class="list2">CSS</li>
-            <li class="list2">Javascript</li>
-          </ul>
-          <h3 class="seeproject">See Project</h3>
+          <div class="grid-container-text">
+            <h2 class="tonic">Uber Navigation</h2>
+            <ul class="grid-list">
+              <li class="canopy-list">Uber</li>
+              <li class="back-list">Lead Developer</li>
+              <li class="year-list">2018</li>
+            </ul>
+            <p class="gridtext">
+              A smart assistant to make driving more safe, efficient, and fun
+              <br />
+              by unlocking your most expensive computer: your car.
+            </p>
+            <ul class="grid-list2">
+              <li class="list2">html</li>
+              <li class="list2">Ruby on rails</li>
+              <li class="list2">CSS</li>
+              <li class="list2">Javascript</li>
+            </ul>
+            <h3 class="seeproject">See Project</h3>
+          </div>
```

Comment:
- [ ] Kindly make sure to implement the hover effect on your buttons to create a good user experience and to also make your design align with that of the Figma design template

![image](https://github.com/CrackedYoda/MyPortfolio/assets/83360636/415eeee6-c7aa-4f10-8ab6-219d33877a14)